src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -2,18 +2,87 @@
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    private static final int DEFAULT_SIZE = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private static final int MULTIPLIER = 2;
+    private int size;
+    private Node<K,V>[] table;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_SIZE];
```

Comment:
It's recommended to use the diamond operator with the correct generic types when initializing the array. Replace with 'new Node<>(DEFAULT_SIZE);' to avoid unchecked assignment warnings.